src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/stratifications/AlleleFrequency.java

```diff
@@ -12,26 +13,74 @@
 /**
  * Stratifies the eval RODs by the allele frequency of the alternate allele
  *
- * Uses a constant 0.005 frequency grid, and projects the AF INFO field value.  Requires
- * that AF be present in every ROD, otherwise this stratification throws an exception
+ * Either uses a constant 0.005 frequency grid, and projects the AF INFO field value or logit scale from -30 to 30.
+ * Requires that AF be present in every ROD, otherwise this stratification throws an exception
  */
+@DocumentedFeature(groupName= "tools", summary = "Stratify by  eval RODs by the allele frequency of the alternate allele") // , extraDocs = {VariantEval.class})
 public class AlleleFrequency extends VariantStratifier {
+
+    public enum StratifyingScale {
+        LINEAR,
+        LOGARITHMIC
+    }
+
+    private StratifyingScale scale;
+    private boolean useCompAFStratifier;
+
+    private static int logLimit = 30; // go from -30 to 30 using logit function
+
     @Override
     public void initialize() {
-        for( double a = 0.000; a <= 1.005; a += 0.005 ) {
-            states.add(String.format("%.3f", a));
+        scale = getVariantEvalWalker().getAFScale();
+        useCompAFStratifier = getVariantEvalWalker().getCompAFStratifier();
+
+        switch (scale) {
+            case LINEAR:
+                for (double a = 0.000; a <= 1.005; a += 0.005) {
+                    states.add(String.format("%.3f", a));
+                }
+                break;
+            case LOGARITHMIC:
+                for (int a = -logLimit; a <= logLimit; a += 1) {
+                    states.add(String.format("%d", a));
+                }
+
         }
     }
 
     public List<Object> getRelevantStates(ReferenceContext referenceContext, ReadsContext readsContext, FeatureContext featureContext, VariantContext comp, String compName, VariantContext eval, String evalName, String sampleName, String FamilyName) {
         if (eval != null) {
             try {
-                return Collections.singletonList((Object)String.format("%.3f", (5.0 * MathUtils.roundToNDecimalPlaces(eval.getAttributeAsDouble("AF", 0.0) / 5.0, 3))));
+                Double alleleFraction  = Collections.max(eval.getAttributeAsDoubleList("AF", 0.0));
```

Comment:
`alleleFraction` -> `alleleFrequency` to minimize confusion (and also because the class is called AlleleFrequency)